stories/helpers.tsx

```diff
@@ -4,6 +4,7 @@ import {select, text, boolean} from '@storybook/addon-knobs'
 import {Table, isNumeric} from '../src'
 import {TABLE} from './data'
 import * as colorSchemes from '../src/constants/colorSchemes'
+import {includes} from 'lodash'
```

Comment:
Weird... `lodash` isn't specified in the `package.json` file. It must be a transitive dependency and somehow importing it works.

Either way, can we use [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) instead?

Reply:
sure!